faker/helpers.go

```diff
@@ -0,0 +1,44 @@
+package faker
+
+import (
+	"math/rand"
+	"strings"
+	"time"
+)
+
+func MustFakeObject(obj interface{}, opts ...Option) interface{} {
+	if err := FakeObject(obj, opts...); err != nil {
+		panic(err)
+	}
+	return obj
+}
+
+func Name() string {
+	var s struct {
+		Name string
+	}
+	MustFakeObject(&s)
+	return s.Name
+}
+
+func Word() string {
+	return Name()
+}
+
+func RandomUnixTime() int64 {
```

Comment:
Similar to `SkipFields`. Are these needed?  Do the tests pass without them?

Reply:
They apparently do, all removed @erezrokah